shell/platform/android/io/flutter/embedding/android/AndroidTouchProcessor.java

```diff
@@ -69,19 +70,28 @@
 
   private static final int _POINTER_BUTTON_PRIMARY = 1;
 
+  private static final Matrix IDENTITY_TRANSFORM = new Matrix();
```

Comment:
Just to confirm since the Android API docs didn't indicate that.  `new Matrix()` is the identity matrix by default?

Reply:
yup

Reply:
https://developer.android.com/reference/android/graphics/Matrix#Matrix()